Makefile

```diff
@@ -75,17 +75,21 @@ install: $(BUILD_DIR)/$(PROJECT)
 cross: $(foreach platform, $(SUPPORTED_PLATFORMS), $(BUILD_DIR)/$(PROJECT)-$(platform))
 
 $(BUILD_DIR)/$(PROJECT)-%: $(STATIK_FILES) $(GO_FILES) $(BUILD_DIR) deploy/cross/Dockerfile
-	GOOS="$(firstword $(subst -, ,$*))" GOARCH="$(lastword $(subst -, ,$(subst .exe,,$*)))" \
+	$(eval os = $(firstword $(subst -, ,$*)))
```

Comment:
its is frustrating, we cant use keys values pairs all though its supported on bash4. 
https://stackoverflow.com/questions/6047648/bash4-associative-arrays-error-declare-a-invalid-option
It would have been much easier to read and less error prone. 

this change looks good to me.